F21/amirsalari/convert.py

```diff
@@ -0,0 +1,123 @@
+import os.path
+import sys
+
+
+def convert(test_word):
+    if(test_word[1] in ["en", "en-US", "en-IE", "en-Latn"]):
+        lowercase_word = test_word[0].lower()
+        return lowercase_word
+
+    # Specification 1
+    if(test_word[1] in ["tr", "az"]):
+        lowercase_word = test_word[0]
+        lowercase_word = lowercase_word.replace('\u0049', '\u0131')  # I ı
```

Comment:
This seems to not be working for some of the Turkish words (test 31 for example). Not sure why!